boards/derfmega256/doc.txt

```diff
@@ -0,0 +1,20 @@
+/**
+ * @defgroup    boards_deRFmega deRFmega128 and deRFmega256 modules from Dresden Elektronik 
```

Comment:
`boards_deRFmega` is defined twice.
Easiest solution would be to define `boards_deRFmega256` and `boards_deRFmega128` instead.